### PR TITLE
Issue 9883 integration test under test com dotcms

### PR DESCRIPTION
**Pull request by erickgonzalez**

### CODE REVIEW

**Review by jdotcms — APPROVED** [review]
good work

File: test/com/dotcms/notifications/BaseContentTypeSystemEventProcessorTest.java [hunk 1]
```diff
@@ -22,6 +25,12 @@
 import static org.junit.Assert.assertTrue;
 
 public class BaseContentTypeSystemEventProcessorTest {
+	
+	@BeforeClass
+	public static void prepare() throws Exception{
+    	//Setting web app environment
+        IntegrationTestInitService.getInstance().init();
```
Comment (dsilvam) [inline test/com/dotcms/notifications/BaseContentTypeSystemEventProcessorTest.java]: This is true unit, no need to call the IntegrationTestInitService

File: test/com/dotcms/util/marshal/MarshalUtilsTest.java [hunk 2]
```diff
@@ -35,6 +38,12 @@
  */
 
 public class MarshalUtilsTest {
+	
+	@BeforeClass
+	public static void prepare() throws Exception{
+		//Setting web app environment
+        IntegrationTestInitService.getInstance().init();
```
Comment (dsilvam) [inline test/com/dotcms/util/marshal/MarshalUtilsTest.java]: This is a true unit test, no need to call IntegrationTestInitService

File: test/com/dotcms/filters/interceptor/jwt/JsonWebTokenInterceptorTest.java [hunk 3]
```diff
@@ -47,6 +49,12 @@
  * @author jsanca
  */
 public class JsonWebTokenInterceptorTest {
+	
+	@BeforeClass
+	public static void prepare() throws Exception{
+    	//Setting web app environment
+        IntegrationTestInitService.getInstance().init();
```
Comment (dsilvam) [inline test/com/dotcms/filters/interceptor/jwt/JsonWebTokenInterceptorTest.java]: Talked to @jdotcms . This is true unit but is not succeeding. Let's leave it as true unit and @jdotcms will fix it.

**Review by dsilvam — CHANGES_REQUESTED** [review]

File: test/com/dotcms/util/marshal/MarshalUtilsTest.java (hunk 2 above)
Comment (erickgonzalez) [inline test/com/dotcms/util/marshal/MarshalUtilsTest.java]: If the IntegrationTestInitService isn't called one test fails: marshalSystemEvent

Comment (dsilvam) [inline test/com/dotcms/util/marshal/MarshalUtilsTest.java]: needs to be fixed as well, @erickgonzalez can you please create issues for this one and JsonWebTokenInterceptorTest. They are true unit and nest to run ok again. Short description just say they are failing

**Review by dsilvam — APPROVED** [review]

**Pull request merged**